### PR TITLE
Add pseudo price to plugin

### PR DESCRIPTION
**Pull request by LiaraAlis**
With this changes it would be possible to save pseudo prices for customer related prices.
The pseudo price will be displayed in the frontend.

Please merge them and publish it with one of the next releases.

Please ask me for questions or if there are any problems with these changes.

### CODE REVIEW

File: Snippets/backend/plugins/user_price/view/prices.ini [hunk 1]
```diff
@@ -10,6 +10,7 @@ prices/any = "Arbitrary"
 prices/header/from = "From"
 prices/header/to = "To"
 prices/header/price = "Price"
+prices/header/pseudoPrice = "Price"
```
Comment (mitelg) [inline Snippets/backend/plugins/user_price/view/prices.ini]: please rename this snippet

File: Bootstrap/Setup.php [hunk 2]
```diff
@@ -130,6 +130,7 @@ private function createTables()
               `articleID` INT(11) NOT NULL DEFAULT '0',
               `articledetailsID` INT(11) NOT NULL DEFAULT '0',
               `price` DOUBLE DEFAULT '0',
+              `pseudoPrice` DOUBLE DEFAULT '0',
```
Comment (mitelg) [inline Bootstrap/Setup.php:142]: please also consider the new column of the table in case of updating the plugin.

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
Hey @LiaraAlis thanks for your PR :+1: 
please have a look at my comments :wink: 
It also seems that your branch is out-dated. please rebase your branch to the current master

File: Bootstrap/Setup.php (hunk 2 above)
Comment (LiaraAlis) [reply Bootstrap/Setup.php:142]: Oh, you're right. I've not considered this. 🤣

Comment (LiaraAlis) [inline Bootstrap/Setup.php]: Changes done. Hope they are okay. I have forked the repository today, so I can not see that my branch is out-dated.

**Comment by LiaraAlis** [comment]
Changes done. Hope they are okay. I have forked the repository today, so I can not see that my branch is out-dated.

**Review by LiaraAlis — COMMENTED** [review]
Changes done. Hope they are okay. I have forked the repository today, so I can not see that my branch is out-dated.

**Review by ghost — APPROVED** [review]

File: Bootstrap/Setup.php (hunk 2 above)
Comment (bpanet) [inline Bootstrap/Setup.php]: A helper method for this is implemented now.

**Comment by mitelg** [comment]
hey @LiaraAlis thanks for your changes, but unfortunately the sync with our interal repo was not working correctly. sorry for the inconvenience! could you please rebase your branch now?

File: Bootstrap/Setup.php [hunk 3]
```diff
@@ -86,6 +86,10 @@ public function update($oldVersion)
             $this->addIndexToPriceTable();
         }
 
+        if (version_compare($oldVersion, '2.1.1', '<')) {
```
Comment (mitelg) [inline Bootstrap/Setup.php]: please use `version_compare($oldVersion, '2.1.1', '<=')` because the next version will be 2.1.2

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Bootstrap/Setup.php (hunk 3 above)
Comment (LiaraAlis) [inline Bootstrap/Setup.php]: Oh, I did not pay attention. Sorry. :(

**Review by LiaraAlis — COMMENTED** [review]
Oh, I did not pay attention of the version number. Sorry. :(

**Comment by LiaraAlis** [comment]
I've updated my branch to the current master.

**Review by mitelg — APPROVED** [review]

**Comment by LiaraAlis** [comment]
Any news here? It would be nice if this feature would be added very soon.

**Comment by mitelg** [comment]
hi @LiaraAlis 
as this plugin is not officially supported by us anymore, the ticket has not a high priority in our workflow. but I will have a look, if I could push it a little bit :wink:

**Comment by mitelg** [comment]
hey @LiaraAlis 

Sorry for the late answer!
The corresponding ticket was closed. If this change is still relevant for you, please feel free to create a new PR.

**Pull request closed without merging**